src/pages/go/adding-version-information.mdx

```diff
@@ -0,0 +1,71 @@
+import { Callout, Steps, Tabs, Tab } from "nextra-theme-docs";
+
+# Adding version information to metrics in Go
+
+Autometrics makes it easy to spot versions and commits that introduce errors or latency.
+
+It produces a `build_info` metric and uses labels to expose the version, commit and branch information of your app to Prometheus.
+
+Example:
+
+```prometheus 
+build_info{branch="main",commit="5cc2aa5447907fbf0a7ddd",version="1.0.0"} 1
+```
+
+## Usage
+
+To add version information to your metrics, simply add the `autometrics.BuildInfo` parameter to the `autometrics.Init()` call:
```

Comment:
The `BuildInfo` argument is always mandatory, but you can initialize it to an empty struct (or skip some fields) if you're not interested

```suggestion
To add version information to your metrics, simply fill the `autometrics.BuildInfo` parameter to the `autometrics.Init()` call:
```